packages/compass-indexes/src/components/indexes/indexes.spec.tsx

```diff
@@ -95,7 +95,9 @@ describe('Indexes Component', function () {
       },
     });
     expect(screen.getByTestId('indexes-toolbar')).to.exist;
-    // TODO: actually check for the error
+    expect(screen.getByTestId('indexes-error').textContent).to.equal(
```

Comment:
Drive-by: I spotted the TODO when I audited the new TODOs I'm adding in this PR.